change/@fluentui-react-breadcrumb-65d8fcef-c70f-4b03-ace4-69630ba9af7c.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30135&commitId=ebd639b23847a8880f5907e9e7d4ba4c9a919601&env=prod&clientType=FLUENTUI)


  <details><summary><b>Breadcrumb Converged</b> 20 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Breadcrumb Converged.appearance - Dark Mode.pressed.chromium.png|282| Changed|
|Breadcrumb Converged.appearance - High Contrast.focused.chromium.png|461| Changed|
|Breadcrumb Converged.appearance - Dark Mode.focused.chromium.png|282| Changed|
|Breadcrumb Converged.appearance - High Contrast.hover.chromium.png|461| Changed|
|Breadcrumb Converged.appearance - Dark Mode.hover.chromium.png|282| Changed|
|Breadcrumb Converged.appearance - High Contrast.default.chromium.png|461| Changed|
|Breadcrumb Converged.appearance - RTL.hover.chromium.png|319| Changed|
|Breadcrumb Converged.appearance - Dark Mode.default.chromium.png|282| Changed|
|Breadcrumb Converged.appearance - High Contrast.pressed.chromium.png|461| Changed|
|Breadcrumb Converged.appearance - RTL.pressed.chromium.png|319| Changed|
|Breadcrumb Converged.appearance - RTL.default.chromium.png|319| Changed|
|Breadcrumb Converged.appearance.default.chromium.png|314| Changed|
|Breadcrumb Converged.appearance.hover.chromium.png|314| Changed|
|Breadcrumb Converged.appearance.pressed.chromium.png|314| Changed|
|Breadcrumb Converged.appearance.focused.chromium.png|314| Changed|
|Breadcrumb Converged.size.focused.chromium.png|971| Changed|
|Breadcrumb Converged.size.default.chromium.png|971| Changed|
|Breadcrumb Converged.size.hover.chromium.png|971| Changed|
|Breadcrumb Converged.appearance - RTL.focused.chromium.png|319| Changed|
|Breadcrumb Converged.size.pressed.chromium.png|971| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>